fix from upstream:

2004-08-20  Daniel Jacobowitz  <dan@debian.org>

	* testglue.c: Prototype abort(3) and exit(3).

diff --git a/testglue.c b/testglue.c
index 55a7f92..4ede6eb 100644
--- a/testglue.c
+++ b/testglue.c
@@ -40,6 +40,10 @@
 #define ORIG_ABORT __wrap_abort
 #define ORIG_MAIN __wrap_main
 #endif
+
+extern void abort (void);
+extern void exit (int);
+
 #endif
 
 #ifdef REAL_MAIN
